### PR TITLE
Update Angular package to v8

### PR DESCRIPTION
**Pull request by lyczos**
In this PR I've removed old files (Angular v5):
- packages/angular/.angular-cli.json => packages/angular/angular.json
- packages/angular/examples/basic-site/.angular-cli.json => packages/angular/examples/basic-site/angular.json
- add packages/angular/examples/basic-site/browserslist
- updated dependencies 

Updated to Angular v8:

- @bulder/angular
- Angular Basic Site Example
- Angular Universal  Example

### CODE REVIEW

**Comment by lyczos** [comment]
@steve8708 I don't know if should I remove package.lock's files (i didn't use mono repo and lerna before), so please clarify that :)

File: packages/angular/browserslist [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```
Comment (ca136) [inline packages/angular/browserslist]: Can you update this to support IE 11?

File: packages/angular/examples/basic-site/browserslist [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# https://github.com/browserslist/browserslist#queries
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+not IE 9-11 # For IE 9-11 support, remove 'not'.
```
Comment (ca136) [inline packages/angular/examples/basic-site/browserslist]: same here, add IE 11 support

File: packages/angular/examples/basic-site/src/polyfills.ts [hunk 3]
```diff
@@ -1,49 +1,4 @@
...
-// import 'core-js/es6/regexp';
-// import 'core-js/es6/map';
-// import 'core-js/es6/weak-map';
-// import 'core-js/es6/set';
-
-/** IE10 and IE11 requires the following for NgClass support on SVG elements */
-// import 'classlist.js';  // Run `npm install --save classlist.js`.
-
-/** IE10 and IE11 requires the following for the Reflect API. */
-// import 'core-js/es6/reflect';
```
Comment (ca136) [inline packages/angular/examples/basic-site/src/polyfills.ts]: Think we'll need some of these for IE 11 support.

Comment (lyczos) [reply packages/angular/examples/basic-site/src/polyfills.ts]: @ca136 The polyfills are now handled by Angular CLI (v8+) itself

File: packages/angular/examples/basic-site/browserslist (hunk 2 above)
Comment (lyczos) [reply packages/angular/examples/basic-site/browserslist]: good catch!

**Comment by lyczos** [comment]
@ca136 I've fixed an issue. I've forgotten to bump version of compiler-cli

File: packages/angular/browserslist (hunk 1 above)
Comment (lyczos) [reply packages/angular/browserslist]: done

**Pull request merged**